### ISSUE TITLE
Sundial version is not aligned with the Uploader

### ISSUE THREAD

**Issue by tomerbraun** [tidepool-org/ble-glucose#18]
The uploader is using sundial version 1.7.0 and ble-glucose version 1.6.0.
It causes a conflict between the moment-timezone versions used by the sundial package.
This Conflict prevents from choosing some of the time zones.

**Comment by gniezen** [comment]
This is a great find, thanks @tomerbraun ! Can you give me an example of such a time zone to make it easier to reproduce?

**Comment by tomerbraun** [comment]
@gniezen sure!
You can try to choose US/Pacific-New

**Comment by gniezen** [comment]
US/Pacific-New was removed from the time zone list, which aligns with the IANA time zone database. It should not be used, but I hadn't noticed that it appeared again. Thanks, I'll try to get this fixed!

**Comment by tomerbraun** [comment]
From what I checked, in the MainPage.js component of the uploader, there is functionality that filters timezones that are invalid. This functionality passes null as the timezone.
Instead of filtering the problematic timezone, it should show only the available time zones. 
According to what I saw, the unsupported time zones are caused by the conflict between the sundial versions.

![image](https://user-images.githubusercontent.com/93592037/223788360-7e1111c1-a9c1-4839-bdc8-b3ab2e28bc06.png)
